main.go

```diff
@@ -147,7 +149,7 @@ func main() {
 
 	if debugMode {
 		opts.Debug = true
-		opts.ProviderAddr = "genesys.com/mypurecloud/genesyscloud"
+		opts.ProviderAddr = "registry.terraform.io/mypurecloud/genesyscloud"
 	}
```

Comment:
This was just changed to `genesys.com` by Brian in [this PR](https://github.com/MyPureCloud/terraform-provider-genesyscloud/commit/a96cdae3f2548e9b0d1a58249214a2534c3542f9). You might need to merge dev into this branch.